astilectron.go

```diff
@@ -67,14 +67,16 @@ type Astilectron struct {
 
 // Options represents Astilectron options
 type Options struct {
-	AcceptTCPTimeout   time.Duration
-	AppName            string
-	AppIconDarwinPath  string // Darwin systems requires a specific .icns file
-	AppIconDefaultPath string
-	BaseDirectoryPath  string
-	DataDirectoryPath  string
-	ElectronSwitches   []string
-	SingleInstance     bool
+	AcceptTCPTimeout     time.Duration
+	AppName              string
+	AppIconDarwinPath    string // Darwin systems requires a specific .icns file
+	AppIconDefaultPath   string
+	BaseDirectoryPath    string
+	DataDirectoryPath    string
+	ElectronSwitches     []string
+	SingleInstance       bool
+	SkipAstilectronSetup bool   // If true, the user must handle provisioning and executing astilectron.
+	TCPPort              string // The port to listen on.
```

Comment:
I'd rather make it a `*int`. That way we know when the user has set a value (!= nil) and `0` is a value that can be used.